tests/unit/test_units.py

```diff
@@ -1,11 +1,87 @@
+import numpy as np
+
+
 from openscm.units import unit_registry
 from openscm.units import DimensionalityError, UndefinedUnitError, UnitConverter
 import pytest
 
 
 def test_unit_registry():
-    tq = unit_registry("CO2")
-    assert tq.to("C").magnitude == 12. / 44.
+    CO2 = unit_registry("CO2")
+    assert CO2.to("C").magnitude == 12.0 / 44.0
+
+
+def test_alias():
+    CO2 = unit_registry("carbon_dioxide")
+    assert CO2.to("C").magnitude == 12.0 / 44.0
+
+
+def test_base_unit():
+    # I don't know why this is this way, anyway
```

Comment:
`carbon` should be defined, right? checking...

Reply:
ye I don't get it either..

Reply:
Ahh, I know now: `C = [carbon]` tells Pint, that `C` is of *dimension* `carbon` (like `s` is of dimension `time`). So we could add `carbon` also as an alias for `C` (add `carbon = C`)?

Reply:
If that works, let's go for it?

Reply:
done